coop_membership/models/event_registration.py

```diff
@@ -78,10 +75,10 @@ def get_date_time_meeting(self):
 
             time_end = self.convert_meeting_time(date_end)
 
-            date_time_meeting = '%s %s %s de %s à %s' % (
-                str(weekday), str(date), str(month), str(meeting_time), str(time_end))
+            date_time_meeting = '%s %s %s de %s à %s' % (str(weekday), str(
+                date), str(month), str(meeting_time), str(time_end))
 
-            return unicode(date_time_meeting, "utf-8")
+            return date_time_meeting.encode('utf-8')
```

Comment:
Try to avoid using this hardcoded "format date" methods that completely ignore the user's locale.
This shouldn't be necessary. Can you check where is this function used and try to avoid it?

If it's on email templates, we can rely on `format_date` or `format_datetime` functions instead

Reply:
@ivantodorovich  Yes it is only used in email template might be we can simlpy use `${object.event_id.date_begin_located}` ?

Reply:
:+1: agree, please do it :)